### ISSUE TITLE
Mails not received with "Enable html emails"

### ISSUE THREAD

**Issue by anorlen** [stcr/subscribe-to-comments-reloaded#540]
Hey!

Mails only come through when I send them as plain text. For some reason if "Enable html email"=yes nothing comes through to my gmail. I used the plugin "WP Mail Logging" and confirmed that the mails are sent successfully without error.

I used another plugin "WP Mail SMTP", it has a "Send test email" feature and I confirmed that I could send html mails with that one that arrived safely to my gmail account.

Regards,
Alexander

**Comment by BobaWebDev** [comment]
Hi @anorlen 

Sine the "WP Mail Logging" plugin reports the email as being sent out successfully but it's not arriving, it might be flagged by GMail for some reason.

Did it perhaps end up in the spam folder?

And can you send over the URL to the website, I'll do some tests.

**Comment by anorlen** [comment]
Hi! www.alexandernorlen.com

Yes i checked the spam, nothing there. I also checked that I do not have any filters that delete it.

I noticed, that if I send the mail (with html) to my webmail at bluehost it appears there, and then it gets delivered to my gmail (connected with POP3) without problem.. so only when I send it directly to my gmail it seems it is not accepted for some reason?

**Comment by BobaWebDev** [comment]
Hi @anorlen 

I submitted a comment, after you approve it I'll be able to test the mail functionality. 

I do have one question, did you make any adjustments to the "Notification Message" ( WP admin > StCR > Notifications ) or is it the default one that the plugin comes with.

**Comment by anorlen** [comment]
I have approved it. I might have changed the notification message.. But
only a few words in that case. No structure or html things.

I'll be at my computer soon if you need closer information.

Alexander

**Comment by anorlen** [comment]
No. I didn't change any of the notification messages. Only some wordings for the comment form..

**Comment by anorlen** [comment]
Btw I'm using StCR version 191028 and WP version 5.3

**Comment by BobaWebDev** [comment]
Hi @anorlen 

Tried it out with a GMail and a YahooMail account. But not receiving it on either.

Not really sure what might be happening there. If WP Mail Logging reports it as successfully sent but it does not arrive I'm thinking that the middleman ( the hosting provider ) is blocking it for some reason.

https://my.bluehost.com/hosting/help/email-deliverability-problems

I don't have a hosting with bluehost so can't check it out to see if there's anything in the control panel where you could check what emails are being blocked and for what reason. 

Best to contact their support ( http://www.bluehost.com/hosting/chatlite ) and ask them about it. If they say they are blocked for something we can handle in the plugin we will do that ASAP.
